generator/main.ts

```diff
@@ -20,6 +20,7 @@ import * as mkdirp from 'mkdirp';
 import * as nunjucks from 'nunjucks';
 import * as path from 'path';
 import * as webidl from 'webidl2';
+import snakeCase = require('snake-case');
```

Comment:
Please use import instead of require :)

Reply:
unfortunately, this snake_case module is ```export = and import = require()``` case.
so we have to ```require``` keyword to use this module.
refer to https://www.typescriptlang.org/docs/handbook/modules.html.

do we have to use the other modules for snake_case module to void this case?

Reply:
Ack